### ISSUE TITLE
Problem with else/player~ not working and crashing PD

### ISSUE THREAD

**Issue by jimruxton** [porres/pd-else#2138]
I am using Ubuntu 22.04, PD version 0.55.0 with Else version  1.0.0_rc11 .  else/player~ is not working for me, even the examples in the example folder fail to play. I have successfully used it in the past so I don't know what recently changed. The other strange thing is if I close a patch with else/player~ in it and DSP on PD crashes with this message in the console.
Pd: signal 6
free(): corrupted unsorted chunks

**Comment by porres** [comment]
Can you try plugdata? Does it work for you over there? just to try and debug this.

anyway, [player~] is an abstraction, based on [tabplayer~], does that work for you?

**Comment by jimruxton** [comment]
Yes it works in plugdata. Also I can confirm that tabplayer is not working in vanilla PD for me. Strange isn't it?

**Comment by porres** [comment]
very strange, can you compile ELSE yourself? Should be pretty simple and straightforward, just compiling and testing [tabplayer~] should be enough

**Comment by porres** [comment]
BTW, a new update is coming in a week... let's make sure it works then ;)

**Comment by porres** [comment]
> Yes it works in plugdata.

@timothyschoen  what magic did you do? :)

**Comment by jimruxton** [comment]
I compiled the latest else on Github and it still didn't work for me.

**Comment by jimruxton** [comment]
Curious if i am the only one having this issue or have others reported it as a problem?

**Comment by porres** [comment]
it's you so far @jimruxton , did you have any warnings while compiling?

I'm about to update and would really like to fix this one, hopefully @timothyschoen can help me using some automated debugger or something

**Comment by jimruxton** [comment]
@porres I can't explained what happened but I can happily report that else/player is now working. It also is no longer crashing PD . I can't say if it is something I installed recently that caused this to work again or whether it was a system update. I'm very sorry if I caused undue alarm but it was very strange how it had stopped working and was crashing pd previously with that 'free(): corrupted unsorted chunks' message. I look forward to the update and hopefully that install will go smoothly. Thanks so much  for responding to my calls for help. I wish I could say what happened and what chased the gremlins away. Feel free to close this issue.
